packages/@lwc/template-compiler/package.json

```diff
@@ -44,7 +44,7 @@
     "dependencies": {
         "@lwc/errors": "6.0.0",
         "@lwc/shared": "6.0.0",
-        "acorn": "~8.10.0",
+        "acorn": "8.10.0",
```

Comment:
acorn has a breaking change to its types in 8.11 so pinning for now: https://github.com/acornjs/acorn/issues/1260